src/libraries/Microsoft.Extensions.Primitives/src/StringValues.cs

```diff
@@ -85,6 +85,26 @@ public StringValues(string?[]? values)
             return value.GetArrayValue();
         }
 
+#if NET9_0_OR_GREATER
+        /// <summary>
+        /// Defines an implicit conversion of a given <see cref="StringValues"/> to a string span.
+        /// </summary>
+        /// <param name="value">A <see cref="StringValues"/> to implicitly convert.</param>
+#pragma warning disable CS3006 // Overloaded method 'StringValues.implicit operator ReadOnlySpan<string?>(in StringValues)' differing only in ref or out, or in array rank, is not CLS-compliant.
```

Comment:
This looks like an analyzer bug, see this [sharplab sample](https://sharplab.io/#v2:EYLgtghglgdgPgAQEwEYCwAoTBtCBnPAUzGABsBPEAYQBkBlKgezAAdSoIYAXACi4CcAroQCUAXUyYEAZgAEyWVVkBvTLPXy5CFADZZUVuwDGULrMYtC/CF0b993HkoBuEUsJFqNy2V40aEAHZZABNCADMIQVIuAG5fDH8AXz8E/xl5XX1DKBMzCysbO0yABh5YRVlXd1FU1UT/AOCwyOiuAEJY1JSMJKA==) too.